cfgov/v1/jinja2/v1/layouts/base.html

```diff
@@ -336,6 +336,8 @@
             {% for js in page.media_js %}
                 {% if 'https://' in js %}
                   s.push('{{ js }}');
+                {% elif js.endswith('?staticroot') %}
```

Comment:
This is just a placeholder approach and there's definitely a better way to do this. After #7303 we assume either that a path is its own URL if it contains `https://` or lives in `js/routes/on-demand/` if not. The wagtailcharts block type needs to load its own JS file that is being served out of the static root, so neither of these work. Per @anselmbradford's comment on https://github.com/cfpb/consumerfinance.gov/pull/7303#discussion_r1003426851, maybe this is sufficient motivation to adopt [paths as objects](https://docs.djangoproject.com/en/4.2/topics/forms/media/#paths-as-objects)?